src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -1443,3 +1465,68 @@ DROP TABLE range_table;
 DROP TABLE none;
 DROP TABLE ref;
 DROP TABLE local_table;
+CREATE TABLE tbl_1 (a INT, b INT);
+CREATE TABLE tbl_2 (a INT, b INT);
+CREATE TABLE tbl_3 (a INT, b INT);
+SELECT create_distributed_table('tbl_1', 'a', shard_count:=4);
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT create_distributed_table('tbl_2', 'a', shard_count:=4);
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT create_distributed_table('tbl_3', 'a', shard_count:=4, colocate_with:='NONE');
+ create_distributed_table
+---------------------------------------------------------------------
+
+(1 row)
+
+SELECT colocation_id AS col_id_1 FROM citus_tables WHERE table_name::text = 'tbl_1' \gset
+SELECT colocation_id AS col_id_2 FROM citus_tables WHERE table_name::text = 'tbl_2' \gset
+SELECT colocation_id AS col_id_3 FROM citus_tables WHERE table_name::text = 'tbl_3' \gset
+-- check that tables are colocated correctly
+SELECT :col_id_1 = :col_id_2;
+ ?column?
+---------------------------------------------------------------------
+ t
+(1 row)
+
+SELECT :col_id_1 = :col_id_3;
+ ?column?
+---------------------------------------------------------------------
+ f
+(1 row)
+
+-- check that there are separate rows for both colocation groups in pg_dist_colocation
+SELECT count(*) FROM pg_dist_colocation WHERE colocationid = :col_id_1;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT count(*) FROM pg_dist_colocation WHERE colocationid = :col_id_3;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+DROP TABLE tbl_1, tbl_3;
+-- check that empty colocation group is dropped and non-empty is not
+SELECT count(*) FROM pg_dist_colocation WHERE colocationid = :col_id_1;
+ count
+---------------------------------------------------------------------
+     1
+(1 row)
+
+SELECT count(*) FROM pg_dist_colocation WHERE colocationid = :col_id_3;
+ count
+---------------------------------------------------------------------
+     0
```

Comment:
should we check whether it's gone on workers as well?

Reply:
Sure